contracts/PLCRFactory.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.4.20;
+
+import "tokens/eip20/EIP20.sol";
+import "./PLCRVoting.sol";
+import "./ProxyFactory.sol";
+
+contract PLCRFactory {
+
+  event newPLCR(address creator, EIP20 token, PLCRVoting plcr);
```

Comment:
does this emit the entire contract? 

is it more efficient to emit the address?
```sol
event newPLCR(address creator, address token, address plcr);
```